stl/inc/filesystem

```diff
@@ -19,12 +19,8 @@
 #include <list>
 #include <locale>
 #include <memory>
-#include <string_view>
-#include <system_error>

```

Comment:
I don't like removing these `include`s; no, they're not necessary anymore because they're gotten through chrono, but we still use them in this file.

Reply:
I agree, let's keep them.

Reply:
I'd prefer not to pay the cost of including `<string_view>`. It's not transitively included in C++17 mode and is not needed, since it simply checks `_HAS_CXX17` (which `<filesystem>` already checks) and includes `<xstring>` (which `<filesystem>` already includes).

It might make sense to include `<system_error>`, `<xfilesystem_abi.h>` and `<xstring>` and avoid transitive inclusion, but `<filesystem>` relies on transitive inclusion anyway. For example, It uses `iterator_traits` without directly including `<iterator>` or `<__msvc_iter_core.h>`.

Reply:
You can include `<xstring>` instead if you want, but the cost of including a header twice is minimal - this is something that compilers have optimized _a lot_.

Reply:
oh oops I just read what you said, nvm.